### PR TITLE
adds reflection gist link

### PR DESCRIPTION
**Pull request by NicholasJacques**
Still working on the rest of the submission. I'll have it done by the end of day today.

### CODE REVIEW

**Comment by NicholasJacques** [comment]
Did Bubble Sort and Insertion Sort without referencing any outside materials.
Had to look up a solution to merge sort.

Refactored bubble sort about 30 times to try to make it as performant as possible.

@case-eee @neight-allen @tmikeschu

File: 1701-b/4module/intermission_work.md [hunk 1]
```diff
@@ -67,8 +67,8 @@
 
 ## Nick Jacques
 
-* Reflection Gist: 
-* Sorting Suite Repo: 
+* Reflection Gist: [link](https://gist.github.com/NicholasJacques/6e0ce459848be5b1ed6a0e6aaa17bbee)
+* Sorting Suite Repo: [link](https://github.com/NicholasJacques/sortingSuiteJS)
```
Comment (case-eee) [inline 1701-b/4module/intermission_work.md]: sorting suite is looking good 👍

File: 1701-b/4module/open_source_proposals/nick_jacques.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
-### Pitch 1: 
+### Pitch 1: [Ecosystem Climate Regulation Services Calculator](https://github.com/rubyforgood/ghgvc)
 
-* **Why:** 
-* **What:** 
-* **I Need:** 
-* **When I'm Done:** 
+* **Why:** I'm interested in conservation and this project is a good opportunity to work on a brownfield project.
+* **What:** Most of the issues look pretty approachable for my skill level with rails. In particular [this](https://github.com/rubyforgood/ghgvc/issues/65) issue looks approachable and worthwhile as well as provides the opportunity to work closely with the project's maintainers to determined what needs to be cleaned up.
+* **I Need:** To communicate with the opener of this issue and nail down exactly what needs to be cleaned since the issue is pretty vague.
```
Comment (case-eee) [inline 1701-b/4module/open_source_proposals/nick_jacques.md]: love the idea of collaborating with the maintainer to clean things up. the issue is pretty vague - i suggest commenting on the issue, introducing yourself, and asking what you can do 👍

File: 1701-b/4module/open_source_proposals/nick_jacques.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 
-### Pitch 2: 
+### Pitch 2: [PublicLab.org](https://github.com/publiclab/plots2)
 
-* **Why:** 
-* **What:** 
-* **I Need:** 
-* **When I'm Done:** 
+* **Why:** Another environmental project that is very actively maintained and has plenty of achievable issues to solve. They also have some issues tagged "first timers only" which is encouraging. This is my favorite project of the three.
+* **What:** Don't have a particular issue in mind. First timer issues get snapped up pretty quick so I could probably just watch the repo and grab one when it comes up.
```
Comment (case-eee) [inline 1701-b/4module/open_source_proposals/nick_jacques.md]: love the idea of working on a first timer issue. you could even reach out to the maintainer and express your interest and maybe they can help point you in the right direction or they may even have something in mind!

**Pull request merged**